app/_how-tos/configure-cyberark-as-a-vault-backend.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Configure CyberArk Conjur as a vault backend
+content_type: how_to
+description: "Learn how to reference CyberArk Conjur secrets from {{site.base_gateway}}."
+products:
+    - gateway
+
+related_resources:
+  - text: Secrets management
+    url: /gateway/secrets-management/
+
+works_on:
+    - on-prem
+
+min_version:
+  gateway: '3.11'
+
+entities: 
+  - vault
+
+tags:
+    - secrets-management
+    - security
+    - cyberark-conjur
+search_aliases:
+  - Conjur
+tldr:
+    q: How can I access CyberArk Conjur secrets in {{site.base_gateway}}? 
+    a: |
+      Configure a Vault entity with `config.auth_method: api_key`, your Conjur endpoint URL (`config.endpoint_url`), account name (`config.account`), login (`config.login`), and API key (`config.api_key`). Reference the secret like `{vault://conjur-vault/BotApp%2FsecretVar}`, assuming your Vault prefix is `conjur-vault` and your secret was stored as `BotApp/secretVar`.
+
+tools:
+    - deck
+
+prereqs:
+  inline: 
+    - title: CyberArk Conjur
+      content: |
+        To run this tutorial, you'll need CyberArk Conjur running with a secret stored. 
+        
+        If you don't already have CyberArk Conjur, you can follow the Docker quickstart guide to [setup an OSS environment](https://www.conjur.org/get-started/quick-start/oss-environment/), [define a policy](https://www.conjur.org/get-started/quick-start/define-policy/), and [store a secret](https://www.conjur.org/get-started/quick-start/store-secret/). 
+
+        {:.warning}
+        > If you're running Conjur in Docker, change the proxy ports in `docker-compose.yml` to `"9443:443"`. The {{site.base_gateway}} and Conjur containers must also be using the same Docker network.
```

Comment:
How do we know that they're using the same network? I didn't find anywhere in the setup for Conjur that told me to specify a network to run on.